docs/usage/index.md

```diff
@@ -21,15 +21,16 @@ Redis don't access the same data, unless this is intended. Whenever two or more
 must be _initialized_ with the same configuration.
 
 ```javascript
-const { FeatureToggles } = require("@cap-js-community/feature-toggle-library");
+const FeatureToggles = require("@cap-js-community/feature-toggle-library").FeatureToggles;
```

Comment:
```suggestion
const { FeatureToggles } = require("@cap-js-community/feature-toggle-library");
```

Reply:
That was a good point. Neither version would have worked. The problem is that static fields and functions are not exposed on instances in JS by design.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static

The closest we can get is to do
```
const { constructor: FeatureToggles } = require("@cap-js-community/feature-toggle-library");
```